2021-01-12 15:21:19,472 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:21:24,644 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:21:29,687 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:21:34,732 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:21:39,775 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:21:58,827 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:22:03,877 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:32:52,665 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:32:57,738 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:33:02,771 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:34:25,577 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:34:30,619 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:34:40,328 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:34:45,385 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:34:50,441 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:35:59,772 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:36:04,836 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:51:45,571 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:51:50,655 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:51:55,691 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:52:57,323 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:53:02,386 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:54:04,021 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
2021-01-12 15:54:09,051 com_data.py[line:36] ERROR {'串口连接异常：': SerialException("could not open port '/dev/stc': FileNotFoundError(2, '系统找不到指定的路径。', None, 3)")}
